src/DGtal/geometry/tools/QuickHullKernels.h

```diff
@@ -0,0 +1,805 @@
+    using Base::on;
+
+    /// The precision as the common denominator for all rational points.
+    double precision;
+    
+    /// Constructor with specified precision
+    ///
+    /// @param[in] aPrecision the chosen precision as the common
+    /// denominator of all rationals (by defaut, 1024).
+    DelaunayRationalKernel( double aPrecision = 1024. )
+      : precision( aPrecision ) {}
+
+    /// @return 'true' if a kernel may induce infinite facets. This is
+    /// typically the case of a Delaunay computation kernel, which
+    /// casts points in higher dimension.
+    bool hasInfiniteFacets() const
+    { return true; }
+
+    /// @param[in] hs an half-space corresponding to a facet.
+    ///
+    /// @return 'true' if the facet associated to this half-space
+    /// correspond to an infinite facet.
+    bool isHalfSpaceFacetInfinite( const HalfSpace& hs ) const
+    {
+      return hs.internalNormal()[ dimension - 1 ] >= 0;
+    }
+
+    /// Transforms a range \a input_points of input points to a range
+    /// \a processed_points of points adapted to a processing by
+    /// QuickHull convex hull algorithm. Keep the mapping information
+    /// between input and processed points. This kernel transforms
+    /// `dim`-dimensional into `dim+1`-dimensional points, where the
+    /// last coordinate lies on a paraboloid. This is the classical
+    /// way for computing a Delaunay triangulation as the convex hull
+    /// of the points onto a paraboloid of higher dimension.
+    ///
+    /// @tparam InputPoint any model of point whose components are
+    /// convertible to Scalar.
+    ///
+    /// @param[out] processed_points the range of points prepared for
+    /// a process by QuickHull.
+    /// 
+    /// @param[out] input2comp the surjective mapping between
+    /// the \a input_points range and the \a processed_points range
+    /// used for computation.
+    ///
+    /// @param[out] comp2input the injective mapping between the 
+    /// \a processed_points range used for computation and the \a
+    /// input_points range.
+    ///
+    /// @param[in] input_points the range of input points.
+    ///
+    /// @param[in] remove_duplicates when 'true', this method removes possible
+    /// duplicates in \a input_points and \a processed_points may thus
+    /// be of smaller size, otherwise, when 'false', it means that
+    /// there are no duplicates in \a input_points.
+    template < typename InputPoint>
+    void makeInput( std::vector< Point >& processed_points,
+                    IndexRange& input2comp, IndexRange& comp2input,
+                    const std::vector< InputPoint >& input_points,
+                    bool remove_duplicates )
+    {
+      const auto F = [&] ( InputPoint input ) -> Point
+        {
+          Point p;
+          Scalar z = 0;
+          for ( Dimension i = 0; i < dimension - 1; i++ ) {
+            const Scalar x = static_cast< Scalar >( round( input[ i ] * precision ) );
```

Comment:
cf earlier comment (and I will skip the next ones)

Reply:
Doc added here and in the kernel class to make more explicit the conversions.